test/intl402/PluralRules/prototype/select/notation.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-pluralruleselect
+description: Checks that the notation option is used appropriately in the select method.
+info: |
+  PluralRuleSelect (
+    _locale_: a language tag,
+    _type_: *"cardinal"* or *"ordinal"*,
+    _notation_: a String,
+    _s_: a decimal String,
+  ): *"zero"*, *"one"*, *"two"*, *"few"*, *"many"*, or *"other"*
+    ...
+    The returned String characterizes the plural category of _s_ according to _locale_, _type_ and _notation_.
+    ...
+---*/
+
+assert.sameValue(new Intl.PluralRules('sl', { notation: 'compact' }).select(1.00000020e6), 'one', 'compact notation');
+assert.sameValue(new Intl.PluralRules('sl', { notation: 'standard' }).select(1.00000020e6), 'other', 'standard notation');
```

Comment:
Curious about these values as well - trying to validate our implementation in Ladybird.

For `notation=standard`, since that's effectively the default value, wouldn't the result be "few"? Chrome, Firefox, Safari and Ladybird all agree on:
```js
>> new Intl.PluralRules('sl').select(1.00000020e6)
"few" 
```

and ICU continues to give us "few" after setting the specific notation.

ICU is also giving us "other" for `notation=compact`, regardless of whether I use a compact display of short or long.